src/main/resources/schema.sql

```diff
@@ -0,0 +1,8 @@
+create table PRODUCTS
+(
+    ID       int              not null AUTO_INCREMENT,
+    NAME     varchar(100)     not null,
+    PRICE    double not null,
+    IMAGE_URL varchar(500),
+    PRIMARY KEY (ID)
+);
```

Comment:
```suggestion
CREATE TABLE products
(
    id        INT NOT NULL auto_increment,
    name      VARCHAR(100) NOT NULL,
    price     DOUBLE NOT NULL,
    image_url VARCHAR(500),
    PRIMARY KEY (id)
);
```

Again, conventions and formatting for professionalism 😎

Reply:
DONE thank you :)